### DISCUSSION TITLE
Add `primaryShade` variants for the other colors

### DISCUSSION THREAD

**Discussion by LeunensMichiel** [mantinedev/mantine#1345] [Feature requests]
Since version `4.2.0` we have access to the `primaryShade` prop which improves default theming out of the box, hands down. Now that we have that logic for the `primaryColor`, I would love to see this implemented for the other color variants.

For example: our design systems implements a base color of shade 5 and we add 4 shades and 5 tints for every color: 
![Shade](https://i.imgur.com/QB52oWS.png)

The underlying styling system chooses an error color that's very dark. Would look better if we can set our main color like in `primaryShade`.
![Form](https://i.imgur.com/N3wzcUE.png)

**Comment by rtivital** [comment]
Colors handling will be changed in 5.0, currently it is hard to implement anything like that without breaking changes

**Reply by LeunensMichiel** [reply]
Nice! Looking forward to V5 then!
